src/NodaTime/Instant.cs

```diff
@@ -502,11 +502,12 @@ public static Instant Max(Instant x, Instant y)
         #endregion
 
         /// <summary>
-        /// Constructs a <see cref="double"/> representing the same instant of time as this value.
+        /// Returns the Julian Date of this instance - the number of days since
+        /// <see cref="NodaConstants.JulianEpoch"/> (noon on January 1st, 4713 BCE in the Julian calendar).
         /// </summary>
-        /// <returns>Returns a <see cref="double"/> containing the number of whole and fractional days since the Julian Epoch (Noon January 1st, 4713 BCE)</returns>
+        /// <returns>The number of days since the Julian Epoch.</returns>
```

Comment:
Fractional sounds like it would be in the range (-1, 1) to me. Have changed to "The number of days (including fractional days)..."